### PR TITLE
Feature/disable apps fall 2017

### PR DESCRIPTION
**Pull request by aaronte**
### Added
* Binary answers to student application details

### Changed
* Enabling toggling of student applications through JS
    * This option can be toggled through changing the value `isStudentApplicationActive` in `config.ts`
    * Opted for a code modification right now vs a admin dashboard toggle since this is much safer to track currently and less destructive (i.e. someone accidentally turning the applications on or off when it doesn't need to be)

### Fixed
* `status` dropdown in applications list not reflecting correct value

### CODE REVIEW

File: src/app/student-application-banner/student-application-banner.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<a id="student-application-banner" class="u-display-block" href="/students/application">
+<a *ngIf="studentAppService.isApplicationActive" id="student-application-banner" class="u-display-block" [routerLink]="['/students/application']">
   <span>Applications to join the team are now open, apply here!</span>
   <img src="assets/icons/tada-emoji.png">
-</a>
+</a>
```
Comment (rperryng) [inline src/app/student-application-banner/student-application-banner.component.html]: [Hmm](https://github.com/uwblueprint/website/pull/153/files#diff-1e70daafb475c0ce3fef7d2728279182R9) 😉 :

Comment (aaronte) [inline src/app/student-application-banner/student-application-banner.component.html]: @rperryng woops!!! Will fix!!!

**Comment by aaronte** [comment]
**Disabled Applications**

localhost-4200-admin-student-details--kpoprwlgr-t2puziciv laptop with mdpi screen 1

**Comment by aaronte** [comment]
**Binary Questions to Admin**

localhost-4200-admin-student-details--kpoprwlgr-t2puziciv laptop with mdpi screen

**Pull request merged**